### PR TITLE
Add prepare for machines upgrade

### PR DESCRIPTION
**Pull request by liangxia**

### CODE REVIEW

**Comment by jhou1** [comment]
@liangxia That is a good solution! Just one question, does the prepare steps need case mapping in polarsion?

**Comment by liangxia** [comment]
> Just one question, does the prepare steps need case mapping in polarion?

@jhou1  I'm not quite sure.  So need @geliu2016 to review/answer.

File: features/upgrade/machines/machines.feature [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Feature: Machine-api components upgrade tests
+  @upgrade-prepare
+  Scenario Outline: Cluster operator should be available after upgrade - prepare
+    Given the expression should be true> "True" == "True"
```
Comment (pruan-rht) [inline features/upgrade/machines/machines.feature]: @liangxia can you please explain what this check is for?

Comment (liangxia) [reply features/upgrade/machines/machines.feature]: @pruan-rht  Act as a place-holder.

Comment (pruan-rht) [reply features/upgrade/machines/machines.feature]: @liangxia placeholder for what?

Comment (liangxia) [reply features/upgrade/machines/machines.feature]: @pruan-rht  According to our upgrade workflow, we need an upgrade-prepare and upgrade-check for each scenario. But some of them do not need any prepare steps, which lead to errors "can not find scenarios" in the log, so we just add a simple/useless step here to get rid of the errors in the log.

Comment (pruan-rht) [reply features/upgrade/machines/machines.feature]: ok, I see, perhaps we should add this note into the PR

Comment (liangxia) [reply features/upgrade/machines/machines.feature]: @pruan-rht  Added the note into the PR.

**Pull request merged**